src/amber/cli/templates/app/config/initializers/client_reload.cr

```diff
@@ -0,0 +1 @@
+Amber::Support::ClientReload.new if Amber.settings.auto_reload?
```

Comment:
💯

Reply:
I disagree.  This is not really core to the framework and is an add-on.  It should be easily removed.  To your point, maybe the setting should be moved to the custom settings?